run_webcore.sh

```diff
@@ -1,9 +1,245 @@
+REPO_ROOT="/media/Storage/McGill/MastersThesis/TouchCORE-2022-03"  # absolute path to the root of the repositories
```

Comment:
Change this as needed. Ideally, this should be the `TOUCHCORE_SRC` environment variable